packages/core/src/blocks/defaultBlockHelpers.ts

```diff
@@ -90,6 +93,27 @@ export const defaultBlockToHTML = <
     );
   }
 
+  // TODO: This is obviously pretty hacky - will need to revisit this when we
```

Comment:
minor feedback; I read "TODO" as something that needs to be fixed before merging.

I would rephrase to first explaining "what's happening" and then adding a note for future recommendations (instead of immediately going ahead and "apologizing" so to say). Suggestion:

> when exporting to external HTML, we convert from `data-` attributes to `style` properties which can be understood by external applications.
> Note: this is a bit hacky to do this here as we're hardcoding this for some default props. We should revisit this when we migrate internal blocks to use the BlockNote API